tests/conftest.py

```diff
@@ -14,9 +14,14 @@
 
 """Py.test hooks."""
 
+import logging
+
 from oauth2client import _helpers
 
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
so that if a test fails, all of the logs will show up in pytest's captured output.